pallets/operation/src/lib.rs

```diff
@@ -72,6 +72,7 @@ pub mod pallet {
         type MinimumBurnedDPR: Get<BalanceOf<Self>>;
         type CreditInterface: CreditInterface<Self::AccountId, BalanceOf<Self>>;
         type UserPrivilegeInterface: UserPrivilegeInterface<Self::AccountId>;
+        type NpowMintDayLimit: Get<BalanceOf<Self>>;
```

Comment:
use storage_value ,and make it value can only be modify by root

Reply:
fixed, please take a look again(for simplicity I reused the weights).